db/db.go

```diff
@@ -0,0 +1,23 @@
+package db
+
+import (
+	"time"
+)
+
+type CronJob struct {
+	ID       string
+	IsActive bool
+	Function string
+	Workload string
+	CronTime string
+	TimeZone string
+	Created  time.Time
+}
+
+type DB interface {
+	GetDistinctActiveJobs() ([]string, error)
+	GetJobDetails(job string) ([]CronJob, error)
```

Comment:
function should be somewhere in this name, e.g. `JobsForFunction(function string)`